css/elements/titles.css

```diff
@@ -0,0 +1,16 @@
+h1 {
+    font-size: 70px;
+    font-weight: inherit;
+    letter-spacing: -4px;
+    line-height: 80px;
+}
+
+/* For Mobile View */
+@media screen and (max-width: 360px) {
+    h1 {
+        font-size: 28px;
+        font-weight: inherit;
```

Comment:
good job on finding a common UI for the h1, but, this line is unnecessary, as the first block already defines the same, therefore, this block is supposed to inherit the same prop

Reply:
Good catch ! Thanks